modules/internal/DefaultRectangular.chpl

```diff
@@ -43,6 +43,9 @@ module DefaultRectangular {
   config param disableArrRealloc = false;
   config param reportInPlaceRealloc = false;
 
+  //TODO make this a param,  it is const only for testing
+  config const parallelAssignThreshold = 2*1024*1024;
```

Comment:
I think we should make this a `param` before merging so that it doesn't show up in the `--help` output